sympy/functions/special/tests/test_spec_polynomials.py

```diff
@@ -78,7 +78,7 @@ def test_jacobi():
     raises(ValueError, lambda: jacobi(-2.1, a, b, x))
     raises(ValueError, lambda: jacobi(Dummy(positive=True, integer=True), 1, 2, oo))
 
-    assert jacobi(n, a, b, x).rewrite("polynomial").dummy_eq(Sum((S.Half - x/2)
+    assert jacobi(n, a, b, x).rewrite(Sum).dummy_eq(Sum((S.Half - x/2)
```

Comment:
If we're keeping the `rewrite("polynomial")` for backwards compatibility (for now) then it should still be tested.